app/addons/documents/doc-editor/components.react.jsx

```diff
@@ -143,12 +143,14 @@ define([
     },
 
     render: function () {
+      var saveButtonLabel = (this.props.isNewDoc) ? 'Create Document' : 'Save Changes';
```

Comment:
minor: no parens needed, but feel free to leave it like it is, no hard opinions here from my side